webpack.tasks/uglify.plugin.js

```diff
@@ -0,0 +1,18 @@
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
How about using [terser-webpack-plugin](https://github.com/webpack-contrib/terser-webpack-plugin) as webpack@5 will be using the terser plugin by default?

Reply:
Yup, it's inside the todolist ;)